cmake/cmaize/package_managers/cmake/cmake_package_manager_impl_/generate_target_config.cmake

```diff
@@ -0,0 +1,146 @@
+if(TARGET ${__gtc_namespace}${__gtc_target_name})
+return()
+endif()")
+    string(APPEND __gtc_file_contents "\n\n")
+
+    string(APPEND
+        __gtc_file_contents
+        "get_filename_component(PACKAGE_PREFIX_DIR "
+        "\"\${CMAKE_CURRENT_LIST_DIR}/../../..\" ABSOLUTE)\n"
+    )
+
+    BuildTarget(GET "${__gtc_tgt_obj}" __gtc_dep_list depends)
+    CXXTarget(GET "${__gtc_tgt_obj}" __gtc_cxx_std cxx_standard)
+    CMaizeTarget(get_property "${__gtc_tgt_obj}" __gtc_version VERSION)
+    CMaizeTarget(get_property "${__gtc_tgt_obj}" __gtc_so_version SOVERSION)
+
+    string(APPEND
+        __gtc_file_contents
+        "
+# Create imported target ${__gtc_namespace}${__gtc_target_name}
+add_library(${__gtc_namespace}${__gtc_target_name} SHARED IMPORTED)
+
+set_target_properties(${__gtc_namespace}${__gtc_target_name} PROPERTIES
+INTERFACE_COMPILE_FEATURES \"cxx_std_${__gtc_cxx_std}\"
+INTERFACE_INCLUDE_DIRECTORIES \"\${PACKAGE_PREFIX_DIR}/include\"
+INTERFACE_LINK_LIBRARIES "
+    )
+
+    set(__gtc_interface_link_libraries)
+    foreach(__gtc_dep_i ${__gtc_dep_list})
+        CMakePackageManager(GET "${self}" __gtc_dep_map dependencies)
+        cpp_map(KEYS "${__gtc_dep_map}" __gtc_keys)
+        cpp_map(GET "${__gtc_dep_map}" __gtc_dep_obj "${__gtc_dep_i}")
+
+        if("${__gtc_dep_obj}" STREQUAL "")
+            continue()
+        endif()
+
+        Dependency(GET
+            "${__gtc_dep_obj}" __gtc_dep_find_tgt_name find_target
+        )
+        list(APPEND __gtc_interface_link_libraries ${__gtc_dep_find_tgt_name})
+    endforeach()
+
+    string(APPEND
+        __gtc_file_contents
+        "\"${__gtc_interface_link_libraries}\"
+)\n"
+    )
+
+    # Based on the shared library suffix, generate the correct versioned
+    # library name and soname that CMake will install
+    if ("${CMAKE_SHARED_LIBRARY_SUFFIX}" STREQUAL ".so")
+        set(__gtc_libname_w_version
+            "lib${__gtc_target_name}.so.${__gtc_version}"
+        )
+        set(__gtc_soname "lib${__gtc_target_name}.so.${__gtc_so_version}")
+    elseif("${CMAKE_SHARED_LIBRARY_SUFFIX}" STREQUAL ".dylib")
+        set(__gtc_libname_w_version
+            "lib${__gtc_target_name}.${__gtc_version}.dylib"
+        )
+        set(__gtc_soname
+            "lib${__gtc_target_name}.${__gtc_so_version}.dylib"
+        )
+    else()
+        string(APPEND __gtc_msg "Shared libraries with the")
+        string(APPEND __gtc_msg "${CMAKE_SHARED_LIBRARY_SUFFIX} suffix")
+        string(APPEND __gtc_msg "are not supported yet.")
+        cpp_raise(
+            UnsupportedLibraryType
+            "${__gtc_msg}"
+        )
+    endif()
+
+    CMakePackageManager(GET "${self}" __gtc_lib_prefix library_prefix)
+    string(APPEND
+        __gtc_file_contents
+        "
+set(_CMAIZE_IMPORT_LOCATION \"\${PACKAGE_PREFIX_DIR}/${__gtc_lib_prefix}/${__gtc_target_name}/${__gtc_libname_w_version}\")
+
+# Import target \"${__gtc_namespace}${__gtc_target_name}\" for configuration \"???\"
```

Comment:
```suggestion
# TODO: Handle different configurations (Release, Debug, etc.)
# Import target \"${__gtc_namespace}${__gtc_target_name}\" for configuration \"???\"
```

Reply:
I missed this, but I'll put it into my next PR.